cmd/lakefs/cmd/setup.go

```diff
@@ -72,7 +73,8 @@ var initCmd = &cobra.Command{
 
 //nolint:gochecknoinits
 func init() {
-	rootCmd.AddCommand(initCmd)
-	initCmd.Flags().String("user-name", "", "an identifier for the user (e.g. \"jane.doe\")")
-	_ = initCmd.MarkFlagRequired("user-name")
+	rootCmd.AddCommand(setupCmd)
+	rootCmd.Flags().MarkDeprecated("init", "use 'lakefs setup' instead")
```

Comment:
Won't this _prevent_ `init` from working?  We would like `init` to continue to work -- so that existing scripts continue to work until the next version bump -- but just print a deprecation note beforehand.  This provides users with a safe rollover period during which *both* commands will work.
Thanks!